src/reactapp/src/context/App/customer/actions.js

```diff
@@ -52,6 +53,11 @@ export async function ajaxLoginAction(dispatch, userCredentials) {
     const response = await ajaxLoginRequest(dispatch, userCredentials);
     const { errors, data } = response;
 
+    if (config.isProductionMode && typeof window !== 'undefined') {
+      window.dispatchEvent(new Event('reload-customer-section-data'));
+      window.location.reload();
```

Comment:
I don't think the dispatch has much effect if you immediately reload the page. It won't finish before the page is reloaded.
Just a page reload should be enough I think. If you just logged in, customer-section-data will be reloaded on the load anyway.

Reply:
The last time I tried out this, I felt this is loading synchronously 🤔  I added that line of code with a thought "if in case" :)

We can remove it for sure